README.md

```diff
@@ -73,7 +71,7 @@ create table db1.table1 (id int, name string);
 insert into db1.table1 values (1, 'a');
 select * from db1.table1;
 ```
-
+- `env POLARIS_HOST=localhost ./regtests/run.sh` - To run regression tests locally, see more options [here](./regtests/README.md).
```

Comment:
I think it's better to keep the simplest option in the top-level readme, and point to advanced users to a dedicated doc. 
Also this command doesn't only work with a local run but also with a docker run.

Reply:
Agree, the purpose of the readme is to be short and quick to start. For details, users can go on the doc.